ua-snap/nws-drought
# Fuse downloads + update environment

This PR updates the environment to be self-contained and able to run the download scripts + the Jupyter notebook which currently just merges the 3 downloads.  That's going to evolve a fair amount here.

To test it,

 - Destroy existing drought-indicators environment and rebuild it by the instructions in the README.
 - Try running `python scripts/download.py` to get new data
 - Try running `Jupyter notebook` then load the `notebooks/preprocessing.ipynb` and run it in full.  If you can see the merged dataset at the end, all is well!
